stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -343,6 +377,8 @@ ::util::Status BFChassisManager::PushChassisConfig(
   node_id_to_port_id_to_port_config_ = node_id_to_port_id_to_port_config;
   node_id_to_port_id_to_singleton_port_key_ =
       node_id_to_port_id_to_singleton_port_key;
+  node_id_to_port_id_to_sdk_port_id_ = node_id_to_port_id_to_sdk_port_id;
+  node_id_to_sdk_port_id_to_port_id_ = node_id_to_sdk_port_id_to_port_id_;
```

Comment:
should it be `node_id_to_sdk_port_id_to_port_id_ = node_id_to_sdk_port_id_to_port_id; ` ?
(there is one `_` in the end)

Reply:
good catch